### PR TITLE
Desktop version of the portfolio

### PR DESCRIPTION
**Pull request by Olamarx**
Add the Desktop version of this project which I started as a mobile version.
Add CSS properties for styling
Add HTML elements for some contents that I only make visible on the desktop version and hide for the mobile version

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,19 +11,35 @@
...
-          I can help you build a product , feature or website Look through some of my work and experience! If you like
+          I can help you build a product, feature or website Look through some of my work and experience! If you like
           what you see and have a project you need coded, don’t hestiate to contact me.
         </p>
+        </div>
+
+
         <ul class="social-media-link">
           <li>
             <a href="#" class="social-media icon">
```
Comment (omar-muhamad) [inline index.html:45]: - [ ] Please add the right font family and font-weight to all heading text and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| image | image  |

File: index.html [hunk 2]
```diff
@@ -56,17 +72,28 @@ <h2 class="section-header-subtitle">I’m a Software Developer</h2>
...
-        <div class="image-paragraph">
+
+        <!-- <div class="multi-story"> -->
+          <div class="image-paragraph">
           <img class="img-pd" src="./images/bg.png" alt="place holder">
           <div class="multiple-column">
            <h2 class="multi-column-h1">Multi-Post Stories</h2>
            <p class="multi-column-det">A daily selection of privately personalized reads; no accounts or sign-ups required.
             has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
            </p>
```
Comment (omar-muhamad) [inline index.html:92]: - [ ] Please add the right font family and font-weight to all heading text and try to stick as much as possible to the given Figma design.
- [ ] Kindly fic the margin between the head text and the horizontal line.

| Figma Design  | Your Design |
| ------------- | ------------- |
|  image | image |

File: index.html [hunk 3]
```diff
@@ -175,8 +235,9 @@ <h2 class="about-header">About me</h2>
               <button class="button" type="button">Get My Resume</button>
             </div>
           </div>
-            <hr class="separate" />
-            <div>
+         <div class="img-line-div"> <img src="./images/Devider.png" class="divider" alt="divider"> </div>
```
Comment (omar-muhamad) [inline index.html:238]: - [ ] For the `about me` section Please add the right font family and font-weight to all heading text and try to stick as much as possible to the given Figma design.
- [ ] Kindly add the proper margin on the left to match the design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| image | image |

File: index.html [hunk 4]
```diff
@@ -219,7 +280,12 @@ <h5 class="text-white">Skills</h5>
...
-      </section> 
+      </section>
+      
+
+      <div class="form-and-footer-section">
+
+        
       <form action="https://formspree.io/f/xnqwzypz" method="post">
         <ul>
           <li class="form-li">
```
Comment (omar-muhamad) [inline index.html:291]: - [ ] For the form in the desktop version Please fix the width of the form elements and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| image  | image  |

File: style.css [hunk 5]
```diff
@@ -524,3 +564,289 @@ footer ul {
...
+    /* border: 2px solid green; */
+  }
+
+  .logo-right-whole {
+    /* border: 2px solid pink; */
+    text-decoration: none;
+    display: flex;
+    text-align: center;
+  }
+
```
Comment (omar-muhamad) [inline style.css]: - [ ] Please remove all commented (unused) code inside your CSS file to make your code `DRY` and readable.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @Olamarx,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] In the mobile version there is a lift margin starting from `headline` section to the end of the page making all elements off the center so please fix that and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| image | image |

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (Olamarx) [reply index.html:45]: I added the font family and the font-weight to all the heading texts as provided in Figma. I mean the name of the font family and a fallback general name too.
You can confirm in the CSS with the class name section-header-title.

done

File: index.html (hunk 2 above)
Comment (Olamarx) [reply index.html:92]: - [ ]  Added the font family and the font-weight to all the heading texts as provided in Figma. I mean the name of the font family and a fallback general name too.
You can confirm in the CSS with the class name 'my-work'.
my

- [ ]  Added the font family and the font-weight to all the heading texts as provided in Figma. I mean the name of the font family and a fallback general name too.
You can confirm in the CSS with the class name 'multi-column-h1'.
multi

- [ ] I fixed the margin between the header text and the line.
margin

File: index.html (hunk 3 above)
Comment (Olamarx) [reply index.html:238]: - [ ] Added the font family and the font-weight to all the heading texts as provided in Figma. I mean the name of the font family and a fallback general name too.
You can confirm in the CSS with the class name **'about-header'.**
about

- [ ] Added the margin-left as required.
new-about

File: index.html (hunk 4 above)
Comment (Olamarx) [reply index.html:291]: ### I fixed this but as a result of what Microverse said that we should not mind the other full name, that is the reason I did not give it the width.
cssnew

**Comment by Olamarx** [comment]
### For the general comment, there are no borders by the left according to my work and figma
### **Figma Mobile Version**  mobilefigma
### **Figma Desktop Version** desktop


### **My Work Mobile Version**
first

### **My Work Desktop Version**
second

**Review by ShadyShawkat — APPROVED** [review]
## Hi @Olamarx, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**